app/features/recent-list/reducer.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import { CONFERENCE_ENDED, CONFERENCE_JOINED } from '../conference';
+
+import type { RecentListItem } from './types';
+
+type State = {
+    recentList: Array<RecentListItem>;
+};
+
+const DEFAULT_STATE = {
+    recentList: []
+};
+
+/**
+ * Reduces redux actions for features/settings.
+ *
+ * @param {State} state - Current reduced redux state.
+ * @param {Object} action - Action which was dispatched.
+ * @returns {State} - Updated reduced redux state.
+ */
+export default (state: State = DEFAULT_STATE, action: Object) => {
+    switch (action.type) {
+    case CONFERENCE_ENDED:
+        return {
+            ...state,
+            recentList: _updateEndtimeOfLastConference(state.recentList)
```

Comment:
we should use the information of the conference that is passed as a parameter (as I mentioned above). I know that at the moment this will always be the last one, but it's not unthinkable that at some point we'll start a new conference which will cause the previous one to end and I don't want us to rely on ordering.

Reply:
Even if we start a new conference the previous conference will end first. Wouldn't it?
There are no plans for two conferences at a single time?

Reply:
Correct on both accounts. But I don't want to rely on that kind of ordering. Explicit is better than implicit :-)

Reply:
Okay. Updated.